kubespawner/templates/form.html

```diff
@@ -18,8 +18,8 @@ <h3>{{ profile.display_name }}</h3>
           <div>
```

Comment:
I think moving the `</label>` [on L34](https://github.com/jupyterhub/kubespawner/pull/738/files#diff-47559ea38f5c75d2661f1c86c79b031011275298348501fcec8a7d80f23f0dbbR34) to before opening this `<div>` for the properties should be the right fix. Then the label for the profile radio button ends before the labels and controls for the profile options.